.github/workflows/smoke-test.yaml

```
@@ -6,11 +6,35 @@
   pull_request:
     branches:
       - dev
+      - workflow/**
 
 jobs:
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)